site/kubernetes/kube-addon.md

```diff
@@ -270,6 +271,14 @@ TCP connection from 10.32.0.7:56648 to 10.32.0.11:80 blocked by Weave NPC.
 UDP connection from 10.32.0.7:56648 to 10.32.0.11:80 blocked by Weave NPC.
 ```
 
+### <a name="key-points"></a><a name="key-points"></a> Things to watch out for
```

Comment:
nit: Why is the `<a name="key-points"></a>` repeated?

Reply:
it was pasted from `troubleshooting` without going through the conscious mind...